src/Converter/OrderItemUnitLineItemsConverter.php

```diff
@@ -13,7 +13,7 @@
 use Sylius\RefundPlugin\Provider\TaxRateProviderInterface;
 use Webmozart\Assert\Assert;
 
-final class LineItemsConverter implements LineItemsConverterInterface
+final class OrderItemUnitLineItemsConverter implements LineItemsConverterInterface
```

Comment:
One day, I would love to fix also this interface (as same definition for two different classes, which expects a different type of refunds)